subsys/partition_manager/Kconfig

```diff
@@ -80,6 +80,13 @@ if BT_RPMSG_NRF53 && SOC_NRF5340_CPUAPP
 
 DT_CHOSEN_Z_IPC_SHM := zephyr,ipc_shm
 
+config RPMSG_NRF53_SRAM_BASE_ADDRESS
+	hex
+	depends on !PARTITION_MANAGER_ENABLED
+	help
+	  If the partition manager is enabled this option should not be used.
```

Comment:
Of course not, cause it has a `depends on !PARTITION_MANAGER_ENABLED`.

What I wonder is, should this have a default value when partition manager is disabled ?

And maybe the help should describe how this setting is to be used when available, that is when partition manager is disabled, instead of describing what happens when partition manager is enabled.

Reply:
I added the help text instead of a comment, so that the rationale behind the `depends on` was visible for users. However, these changes caused issues with NCS projects which has not defined RPMSG buffers in partition manager. So I had to make more intrusive changes to be able to hide these options.

I have modified the patch now so that the only thing fixed is the bug reported, I will add a task for unifying the rpmsg buffer declaration across the different nRF53 samples.